CMakeLists.txt

```diff
@@ -272,7 +272,7 @@ if (LIBUNWIND_ENABLE_ASSERTIONS)
 
   # On Release builds cmake automatically defines NDEBUG, so we
   # explicitly undefine it:
-  if (uppercase_CMAKE_BUILD_TYPE STREQUAL "RELEASE")
+  if ((uppercase_CMAKE_BUILD_TYPE STREQUAL "RELEASE") AND (NOT RUST_SGX))
```

Comment:
This is to suppress debug prints in release build. 

Actually, Cmake env by default, sets NDEBUG for release build, and the libunwind has compile time checks for NDEBUG for printing or not debug logs. But, they also have a framework of enabling logs through env variables, which they want to retain in release build, hence by default CmakeList undefines the NDEBUG for release builds.

Since, we want to have a compile time switch for debug logs, I thought that not disabling NDEBUG for release build might be the appropriate way. Please let me know if we want to go about this any other way.